intro-HPC/gent/macros.tex

```diff
@@ -3,10 +3,11 @@
 \macro{\university}{UGent}
 \macro{\association}{AUGent}
 % HPC definitions
-\macro{\hpc}{HPC-UGent}
-\macro{\hpcname}{Delcatty}
+\macro{\hpc}{UGent-HPC}
```

Comment:
It reads better in this form, so after this pull request is merged it will look beter then before it was merged, that's the point, we can fix this in https://github.com/hpcugent/vsc_user_docs/issues/87